platform/mellanox/mlnx-platform-api/sonic_platform/fan_drawer.py

```diff
@@ -51,14 +51,15 @@ def get_direction(self):
             return FanBase.FAN_DIRECTION_NOT_APPLICABLE
         
         try:
-            from .fan import FAN_DIR
-            with open(FAN_DIR, 'r') as fan_dir:
-                fan_dir_bits = int(fan_dir.read())
-                fan_mask = 1 << self._index - 1
-                if fan_dir_bits & fan_mask:
+            from .fan import FAN_DIR, FAN_DIR_VALUE_INTAKE, FAN_DIR_VALUE_EXHAUST
+            with open(FAN_DIR.format(self._index), 'r') as fan_dir:
```

Comment:
should we use utils.read_int_from_file to read the content?

Reply:
Fixed